utils/utils.jsx

```diff
@@ -1784,3 +1720,26 @@ export function enableDevModeFeatures() {
         },
     });
 }
+
+export function getSortedUsers(state, reactions, currentUserId, profiles, getDisplayNameFn = getDisplayName) {
```

Comment:
I like the idea here, but this can't be used in the `mapStateToProps` because of how it returns a new object each time like I mentioned above. If we use `displayUsername` instead of `getDisplayName` for this, we can avoid passing the whole state tree which will let us turn this into a selector.